tools/niminst/buildsh.nimf

```diff
@@ -159,7 +159,10 @@ case $ucpu in
       COMP_FLAGS="$COMP_FLAGS -m64"
       LINK_FLAGS="$LINK_FLAGS -m64"
       mycpu=`uname -p`
-	else
+      if [ "$(mycpu)" = "powerpc64le"] ; then
+	mycpu="powerpc64el"
```

Comment:
is this really needed? doesn't uname -p return that exact value (you can see mycpu above defined as uname -p) ?

Reply:
ah wait this is fully redundant :)